### DISCUSSION TITLE
failed to build docker-compose

### DISCUSSION THREAD

**Discussion by parvbarot** [pwndoc/pwndoc#423] [Q&A]
There are some errors I'm getting when I run docker-compose build 

The command '/bin/sh -c npm run build' returned a non-zero code: 1
ERROR: Service 'pwndoc-frontend' failed to build : Build failed

**Comment by Zeecka** [comment]
Duplicate of #383 , there is already a pending PR #384.
You can also check my fork [pwndoc-ng](https://github.com/pwndoc-ng/pwndoc-ng).
